homeassistant/components/velbus/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "velbus",
   "name": "Velbus",
   "documentation": "https://www.home-assistant.io/integrations/velbus",
-  "requirements": ["python-velbus==2.0.35"],
+  "requirements": ["python-velbus==2.0.36"],
   "config_flow": true,
   "dependencies": [],
   "codeowners": ["@cereal2nd"]
```

Comment:
Please correct your github tag.

Reply:
what do you mean by this?
2.0.36 is available on pypi: https://pypi.org/project/python-velbus/

Reply:
Your github tag starts with a capital C, not a lowercase c. Please change to `@Cereal2nd`.

Reply:
ah, the codeowner will fix